cmd/monitorcodegen/main.go

```diff
@@ -32,35 +32,7 @@ func buildOutputPath(pkg *selfdescribe.PackageMetadata) string {
 	return path.Join(outputDir, genMetadata)
 }
 
-// shouldRegenerate determines whether the metadata file needs regenerated based on its existence and timestamps.
-func shouldRegenerate(pkg *selfdescribe.PackageMetadata, outputFile string) (bool, error) {
```

Comment:
Does this not result in a speedup with the new method?

Reply:
The speed of `go generate ./...` is sub-1 second after the first invocation, so I'm not too worried about the speed.